Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG REGISTRY_PREFIX=''
+ARG CODENAME=bionic
+
+FROM ${REGISTRY_PREFIX}ubuntu:${CODENAME} as builder
+
+RUN set -x \
+	&& apt update \
+	&& apt upgrade -y \
+	&& apt install --yes --no-install-recommends \
+		build-essential \
+		autotools-dev \
+		automake \
+		pkg-config \
+                libltdl-dev \
+		ca-certificates \
+		curl \
+		libqt4-dev \
+		libqt4-sql-sqlite \
+		x11-apps 
```

Comment:
x11-apps are not needed to build xca, but to forward X11 when the container runs.